python/ray/train/base_trainer.py

```diff
@@ -583,8 +583,9 @@ def training_loop(self) -> None:
 
         ``self.datasets`` have already been preprocessed by ``self.preprocessor``.
 
-        You can use the :ref:`Tune Function API functions <tune-function-docstring>`
-        (``train.report()`` and ``train.get_checkpoint()``) inside
+        You can use the Tune Function API functions
```

Comment:
These should now be Train utilities? https://docs.ray.io/en/master/train/api/api.html#ray-train-utilities